resources/eventing/charts/controller/templates/pre-upgrade-job.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels: {{- include "eventing.preUpgrade.labels" . | nindent 4 }}
+  name: {{ template "eventing.fullname" . }}-pre-upgrade
+  annotations:
+    helm.sh/hook-weight: "0"
+    helm.sh/hook: "pre-upgrade"
+    helm.sh/hook-delete-policy: "before-hook-creation,hook-succeeded"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels: {{- include "eventing.preUpgrade.labels" . | nindent 4 }}
+  name: {{ template "eventing.fullname" . }}-pre-upgrade
+  annotations:
+    helm.sh/hook-weight: "0"
+    helm.sh/hook: "pre-upgrade"
+    helm.sh/hook-delete-policy: "before-hook-creation,hook-succeeded"
+rules:
+  - apiGroups: ["apps"]
+    resources: ["deployments", "deployments/scale"]
+    resourceNames: ["eventing-controller", "eventing-publisher-proxy"]
+    verbs: ["get", "list", "patch", "delete"]
+  - apiGroups: [""]
+    resources: ["secrets", "configmaps"]
+    verbs: ["get", "list"]
+  - apiGroups: ["eventing.kyma-project.io"]
+    resources: ["subscriptions", "eventingbackends"]
```

Comment:
If this manifest is applied to 1.23, would it fail since we mention `eventingbackends`?

Reply:
The `pre-rel124-kyma-gke-upgrade` test is passing, so this should be fine ??